src/harlequin/config_wizard.py

```diff
@@ -175,6 +175,8 @@ def _prompt_for_path() -> tuple[Path, bool]:
         style=HARLEQUIN_QUESTIONARY_STYLE,
     ).unsafe_ask()
     path = Path(raw_path)
+    path = Path.expanduser(path)
+    path.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
I'd rather this happens later, right before we write the file; I don't want to create an empty directory if the user aborts the config wizard without writing the file.

Reply:
Ah i didnt catch this, let me see what i can do :)

Reply:
i have done some necessary changes which i hope fixes this issue